JOBS.md

```diff
@@ -0,0 +1,55 @@
+# We're Hiring Engineers!
+
+Your primary work will be to maintain and improve Ganache, our "Ethereum Simulator". Ganache is a tool that allows developers to start a local blockchain environment to enable easy, fast, and safe Ethereum development.
+
+Ganache is packaged as a JavaScript library, a command-line application, and a Graphical User Interface. It is downloaded more than 400,000 times every month. You can check out the code here: https://github.com/trufflesuite/ganache
+
+You'll share responsibilities with your team to:
+
+ * Ship regular updates to stay up to date with the changing ecosystem
+ * Fix bugs and otherwise improve the end-user experience
+ * Implement a plug-in architecture to enable the use of user-maintained custom blockchain simulators
+ * Design and implement an interface to allow local modifications to the Ethereum Remote Procedure Call interface as well as the low-level Ethereum Virtual Machine
+ * Invent new and novel ways to improve the developer experience across our suite of tools and the rest of the Ethereum ecosystem.
+ * Implement CI/CD to fully automate our release process
+ * Write and maintain our documentation
+
+ ## About You
+
+You're comfortable with:
+ * Spoken and Written Communication
+ * Architecting solutions to open-ended problems
+ * Writing software and tests; you probably have 4+ years of professional experience
```

Comment:
This would be interpreted as you need 4 years of experience and could filter good candidates

Reply:
A very small part of me wants to think that if a potential candidate with less than four years experience reads this and determines that they can't apply because of it then they might not be as comfortable with written communication as we'd like 😁.

But truth be told, you're probably right, and I do feel that the diversity of applicants could be much greater than it has been, so let's brainstorm ways to improve our job posting (which this was copy+pasted from) to remove potential biases like these! This would be a great topic for the alignment meeting.